tests/components/conftest.py

```diff
@@ -626,6 +631,19 @@ async def _async_handle_step(
         setattr(flow, "__flow_seen_before", hasattr(flow, "__flow_seen_before"))
 
         if result["type"] is FlowResultType.FORM:
+            if step_id := result.get("step_id"):
+                for header in ("title", "description"):
```

Comment:
The documentation https://developers.home-assistant.io/docs/data_entry_flow_index#labels--descriptions says that the `description` for a form  is optional, but it doesn't say anything about the `title`.

Testing shows that the integration name is automatically used for the title if it is missing.